guides/source/action_controller_overview.md

```diff
@@ -830,55 +899,33 @@ class LoginActionCallback
 end
 ```
 
-Again, this is not an ideal example for this action callback, because it's not run in the scope of the controller but gets the controller passed as an argument. The class must implement a method with the same name as the action callback, so for the `before_action` action callback, the class must implement a `before` method, and so on. The `around` method must `yield` to execute the action.
+The above is not an ideal example. The `LoginActionCallback` method is not run in the scope of the controller but gets `controller` as an argument.
```

Comment:
This comment is pretty old. I'm not sure if we can add a better example though.
Maybe replace it with the duration measurement example mentioned earlier?

Reply:
Updated